### PR TITLE
Gimbap commercial component

### PR DESCRIPTION
**Pull request by uplne**
# New commercial component to promote Guardian services
## What is the value of this and can you measure success?

Increased interest in guardian products and membership.
## Does this affect other platforms - Amp, Apps, etc?

Not yet.
## Screenshots

screen shot 2016-03-21 at 09 41 19
screen shot 2016-03-21 at 09 41 33
screen shot 2016-03-21 at 09 41 42
screen shot 2016-03-21 at 09 41 51

screen shot 2016-03-15 at 16 35 31
screen shot 2016-03-15 at 16 35 57
screen shot 2016-03-21 at 09 48 37
screen shot 2016-03-21 at 09 59 23

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/creatives/template.js [hunk 1]
```diff
@@ -54,10 +55,42 @@ define([
...
+        this.params.logomasterclasses = svgs('logomasterclasses');
         this.params.arrowRight = svgs('arrowRight', ['i-right']);
         this.params.logoguardian = svgs('logoguardian');
         this.params.marque36iconCreativeMarque = svgs('marque36icon', ['creative__marque']);
         this.params.logoFeatureLabel = 'Paid for by';
+
+        this.mapGimbap();
+    };
+
+    Template.prototype.mapGimbap = function () {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template.js]: I am not sure about this bit. Any clever ideas how to solve it in more efficient way? @regiskuckaertz

File: static/src/inline-svgs/commercial/logo-jobs-horizontal.svg [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<svg x="0px" y="0px" width="112.8px" height="15.4px" viewBox="0 0 112.8 15.4">
+	<g>
+		<g>
+			<g>
```
Comment (regiskuckaertz) [inline static/src/inline-svgs/commercial/logo-jobs-horizontal.svg]: Couldn't we get rid of a `G` or two here?

File: static/src/inline-svgs/commercial/logo-jobs-horizontal.svg [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+				<path class="logo--part1" d="M74.6,11.5h4.3v-0.7l-0.8-0.4V5.3C78.4,5.1,78.7,5,79.1,5c0.7,0,1.1,0.3,1.1,1.1v4.3l-0.8,0.4v0.7h4.3v-0.7
```
Comment (regiskuckaertz) [inline static/src/inline-svgs/commercial/logo-jobs-horizontal.svg]: You could then move the class to the surrounding `G` element, no?

File: static/src/javascripts/projects/common/modules/commercial/creatives/template.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/template.js]: that is what template-preprocessor is for, you could move this code over there, no?

File: static/src/stylesheets/module/commercial/gimbap/_gimbap.scss [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+        left: calc(50% - 650px);
+    }
+}
+
+.gimbap-wrap {
+    padding: 0 $gs-gutter / 2 $gs-baseline;
+    position: relative;
+
+    @include mq(tablet) {
+        max-width: 740px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/gimbap/_gimbap.scss]: You can remove all the `max-width`, the paddings alone will do (my bad sorry)

File: static/src/stylesheets/module/commercial/gimbap/_gimbap.scss [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+
+        &::before,
+        &::after {
+            width: calc(50% - 650px);
+        }
+    }
+}
+
+.gimbap-wrap__box {
+    border-top: 1px solid colour(news-accent);
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/gimbap/_gimbap.scss]: Could you use generated content to avoid the extra `DIV`?

**Comment by regiskuckaertz** [comment]
So this is why people all over the world call you Stevawesome?!

giphy

File: static/src/stylesheets/module/commercial/gimbap/_gimbap.scss (hunk 5 above)
Comment (uplne) [inline static/src/stylesheets/module/commercial/gimbap/_gimbap.scss]: x0dql

**Comment by regiskuckaertz** [comment]
Let's launch this thing on code and simplify the SVGs later so that we can start testing.

giphy 1

:+1:

**Pull request merged**